inc/saiport.h

```diff
@@ -1456,22 +1458,26 @@ typedef sai_status_t (*sai_clear_port_all_stats_fn)(
  *
  * Passed as a parameter into sai_initialize_switch()
  *
+ * @param[in] switch_id Switch id
  * @param[in] count Number of notifications
  * @param[in] data Array of port operational status
  */
 typedef void (*sai_port_state_change_notification_fn)(
```

Comment:
As per my understanding of the current model, only create time we need the switch-id. Later we only need the OID and OID somehow maps to switch-id. So if the sai_port_oper_status_notification_t  has port OID we should not need switch-id.